Feature/subscriptions (#875)
mercadopago/devsite-docs: guides/online-payments/subscriptions/integration.es.md

`auto_recurring.frequency_type` (requerido) | Indica el tipo de frecuencia. Puede ser por mes (months) o días (days). En conjunto con la frecuencia, definen el ciclo de cuotas que va a tener una suscripción. <br><br>Por ejemplo, si cada quince días se necesita generar una cuota para ser cobrada quedaría de la siguiente forma: `auto_recurring.frequency`: 15 y  `auto_recurring.frequency_type`: days |
`auto_recurring.transaction_amount` | Si indicamos el monto, es fijo. Si no completamos este campo, se entiende como monto variable. Se admite un máximo de dos decimales separados por punto. |
`auto_recurring.currency_id` (requerido) | Identifica la moneda que corresponde al país. |
`auto_recurring.repetitions` | Indica si la suscripción va a tener un límite. Si no se especifica no tiene límite. Este límite se relaciona con `auto_recurring.frequency` y `auto_recurring.frequency_type`. |
`auto_recurring.free_trial.frequency` | Define un período de prueba inicial  y retrasa el primer cobro. Indica la cantidad de tiempo que no se va a cobrar por el servicio en base al tipo de frecuencia. Tiene que ser consistente con `auto_recurring.frequency`. |

> `auto_recurring.free_trial.frequency_type` | Indica la cantidad de cuotas que no se van a cobrar por el servicio. Tiene que ser consistente con `auto_recurring.frequency_type` |

Reviewer: ```suggestion
`auto_recurring.free_trial.frequency_type` | Indica la cantidad de cuotas que no se van a cobrar por el servicio. Tiene que ser consistente con `auto_recurring.frequency_type`. |
```

    "sandbox_init_point": "https://sandbox.mercadopago.com/mlm/debits/new?preapproval_plan_id=2c938084726e18d60172720000000000",
    "auto_recurring": {
        "frequency": 1,
        "frequency_type": "months",
        "transaction_amount": 1100,
        "currency_id": "[FAKER][CURRENCY][ACRONYM]",
        "repetitions": 12,
        "free_trial": {
            "frequency": 1,
            "frequency_type": "months"
        }
    }
}

```
¡Listo! Ya podemos crear la suscripción y asociarla a tu plan.

### Crear suscripción

Una vez generado tu plan y obtenido tu `preapproval_plan_id`, crea la suscripción por API de la siguiente manera:

[[[
```curl curl --location --request POST 'https://api.mercadopago.com/preapproval \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer ENV_ACCESS_TOKEN' \
--data-raw '{
   "preapproval_plan_id":"2c938084726e18d60172720000000000",
   "card_token_id":"9fca87c7338585abd000111222333444",
   "payer_email":"test_user_XXXX@testuser.com"
   
}'
```
]]]

#### Atributos

Atributos |	Descripción
--- | ---
`preapproval_plan_id` (requerido) | Hace referencia al Template o Plan previamente generado.|

> `card_token_id (requerido)` | La información de la tarjeta será convertida en un token para enviar los datos de modo seguro. |

Reviewer 2: ```suggestion
`card_token_id`(requerido) | La información de la tarjeta será convertida en un token para enviar los datos de modo seguro. |
```

> `payer_email (requerido)` | E-mail del pagador. |

Reviewer 2: ```suggestion
`payer_email` (requerido) | E-mail del pagador. |
```

> WARNING
> 
> Importante
> 

> > ¿Tienes dudas sobre cómo crear el token de pago? Encuentra toda la información en la sección de __<a href="https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/sdks/official/js/" target="_blank">Capturar datos de la tarjeta</a>__.

Reviewer: ```suggestion
> ¿Tienes dudas sobre cómo crear el token de pago? Encuentra toda la información en la sección de __<a href="https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/online-payments/checkout-api/receiving-payment-by-card#bookmark_captura_los_datos_de_la_tarjeta" target="_blank">Capturar datos de la tarjeta</a>__.
```

`status` | Estado de la suscripción. Puede ser `pending` o `authorized`. |
`auto_recurring.frequency` | Indica la cantidad de tiempo o ciclo en base al tipo de frecuencia. |
`auto_recurring.frequency_type` | Indica el tipo de frecuencia. Puede ser por mes (months) o días (days). En conjunto con la frecuencia, definen el ciclo de cuotas que va a tener una suscripción.<br><br> Por ejemplo, si cada quince días se necesita generar una cuota para ser cobrada quedaría de la siguiente forma: `auto_recurring.frequency`: 15 y  `auto_recurring.frequency_type`: days |
`auto_recurring.transaction_amount` | Monto que se aplica a la suscripción. |
`auto_recurring.currency_id` | Identifica la moneda que corresponde al país. |
`auto_recurring.end_date` | Indica si la suscripción va a tener un límite. Si no se especifica no tiene límite. |
`auto_recurring.free_trial.frequency` | Define un período de prueba inicial durante el cual podrás retrasar el primer cobro, indica la cantidad de tiempo que no se va a cobrar por el servicio en base al tipo de frecuencia. Tiene que ser consistente con `auto_recurring.frequency`. |

> `auto_recurring.free_trial.frequency_type` | Indica la cantidad de cuotas que no se van a cobrar por el servicio. Tiene que ser consistente con `auto_recurring.frequency_type` |

Reviewer: ```suggestion
`auto_recurring.free_trial.frequency_type` | Indica la cantidad de cuotas que no se van a cobrar por el servicio. Tiene que ser consistente con `auto_recurring.frequency_type`. |
```
